lib/spack/spack/util/module_cmd.py

```diff
@@ -32,7 +51,6 @@ def module(*args, **kwargs):
             stdout=subprocess.PIPE,
             stderr=subprocess.STDOUT,
             shell=True,
```

Comment:
You would wanna exec ["/bin/sh", "-c", script] directly instead of `shell=True`; you never know what python means with shell=True.

Reply:
OK.